.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please delete this file.

Reply:
will try to add it to .gitignore. It's some vscode editor thing